release-notes/major10/10.3.0/valtimo-frontend-libraries.md

```diff
@@ -30,6 +30,19 @@ The following features were added:
   The range option is not available for fields of data type "Yes / no" in case-management.
   If a number field has the value '0', the search form can be submitted in the cases page.
 
+* **Resizable Carbon sidenav**
+
+  After major 10, the product switched over to the Carbon design system. The sidenav included in this design system does
```

Comment:
I would scrap the first sentence, and rephrase the second one to:

The sidenav included in the Carbon design system does not include resize functionality by default.